Can not compile in debug mode

Hello,
I'm trying to track down issue #524, but I'm stuck at trying to compile the project in debug mode. I get the following errors when I try to build with command `./build.sh debug`:
> /usr/bin/ld: cannot find -lwallet_merged
> /usr/bin/ld: cannot find -lepee
> /usr/bin/ld: cannot find -leasylogging

I'm getting many errors like the one below when executing that script.
>CMake Error at external/easylogging++/CMakeLists.txt:53 (install):
>install TARGETS given no LIBRARY DESTINATION for shared library target
>"easylogging".

--- a/get_libwallet_api.sh
+++ b/get_libwallet_api.sh
@@ -108,6 +108,7 @@ elif [ "$BUILD_TYPE" == "debug-android" ]; then
 elif [ "$BUILD_TYPE" == "debug" ]; then
     echo "Building libwallet debug"
     CMAKE_BUILD_TYPE=Debug
+    STATIC=true
 else
     echo "Valid build types are release, release-static, release-android, debug-android and debug"
     exit 1;

